src/index.js

```diff
@@ -1 +1,52 @@
 'use strict';
+
+const { Server } = require('http');
+const fs = require('fs');
+const zlib = require('zlib');
+const path = require('path');
+
+const server = new Server();
+
+server.on('request', (req, res) => {
+  if (req.method === 'POST') {
+    const chunks = [];
+
+    req.on('data', (chunk) => {
+      chunks.push(chunk);
+    });
+
+    req.on('end', () => {
+      const body = Buffer.concat(chunks).toString().split('&');
+      const formData = {};
+
+      body.forEach((chunk) => {
+        const [key, value] = chunk.split('=');
+
+        formData[key] = value;
+      });
+
```

Comment:
<img width="486" alt="Screenshot 2023-10-29 at 18 35 29" src="https://github.com/mate-academy/node_form-data/assets/135232001/a87dfdd6-d3b8-4422-9940-762172b0de6d">
Consider creating a file on the server and saving your data here